operator/pkg/helmreconciler/reconciler.go

```diff
@@ -377,8 +379,29 @@ func (h *HelmReconciler) getOwnerLabels(componentName string) (map[string]string
 	return h.addComponentLabels(labels, componentName), nil
 }
 
-// applyLabelsAndAnnotations applies owner labels and annotations to the object.
-func (h *HelmReconciler) applyLabelsAndAnnotations(obj runtime.Object, componentName string) error {
+// getOwnerReferences
+func (h *HelmReconciler) getOwnerReference(componentName string) (*metav1.OwnerReference, error) {
+	crName, err := h.getCRName()
+	if err != nil {
+		return nil, err
+	}
+	crUID, err := h.getCRUID()
+	if err != nil {
+		return nil, err
+	}
+	ownerRef := &metav1.OwnerReference{
+		APIVersion:         h.iop.ApiVersion,
+		Kind:               h.iop.Kind,
+		Name:               crName,
+		UID:                crUID,
+		BlockOwnerDeletion: pointer.BoolPtr(true),
+		Controller:         pointer.BoolPtr(true),
+	}
```

Comment:
Owner ref is IOP instance, right?

Reply:
I understood by iop instance you meant operator deployment, but I get it now.